backend/src/controllers/question.js

```diff
@@ -6,16 +7,18 @@ const errorHandler = require('./error')
 const { responseHandler, userLikesMapper } = require('../utils/helpers')
 
 /* GET question data. */
-exports.getQuestion = function ({ params }, res) {
+exports.getQuestion = function ({ user, params }, res) {
+    const userID = user && user.id || ANONYMOUS
     const getQuestion = Promise.all([
         questionModel.getQuestion(params.id),
-        likesModel.getLikes({ type: 'question', id: params.id })
-    ]).then(([question, likes]) => {
+        likesModel.getLikes({ type: 'question', id: params.id }),
+        likesModel.getUserLiked({ type: 'question', id: params.id, userID })
```

Comment:
I feel like this is an unneeded additional call if we just got return list of likes in the `.getLikes` call. Then you can deduce the second part trivially.